apps/client/app/dashboard/(user)/(nomination)/NominationBox.tsx

```diff
@@ -18,7 +19,7 @@ import {
 } from '@/components/ui/popover'
 import { mutations } from '@/lib/graphql-queries'
 import { useMutation } from '@apollo/client'
-import usePeerReviews from '@/app/dashboard/(user)/usePeerReviews'
+import usePeerReviews from '@/app/dashboard/(user)/(nomination)/usePeerReviews'
```

Comment:
is this dynamic folders? should they have  this brackets? [ ]

Reply:
Nope, they're not dynamic folders. If you use (parentheses), then they're ignored by the routing

Reply:
sweet